samples/sensor/die_temp_polling/boards/mimxrt1064_evk.conf

```diff
@@ -0,0 +1 @@
+CONFIG_REQUIRES_FULL_LIBC=y
```

Comment:
there's already 4 overrides on this sample board directory that only has this line and none of them explains why, would it make sense to move this to the sample prj.conf? Or as a driver dependency if it depends of the driver, right now it's unclear

Reply:
Good point. 
I checked and it's not required to run this sample (at least on the RT 1064) so I removed the board configuration. 
I would not remove it from the other boards as I'm unsure about the specific dependencies.